shell/platform/embedder/embedder.h

```diff
@@ -388,6 +387,10 @@ typedef void (*FlutterPlatformMessageCallback)(
     const FlutterPlatformMessage* /* message*/,
     void* /* user data */);
 
+typedef void (*FlutterDataCallback)(const void* /* data */,
+                                    size_t /* size */,
+                                    void* /* user data */);
```

Comment:
This should really be commented both here and on the construction method; it would be easy to miss that this `user data` isn't the same as the parameter with the same comment on most other callbacks in this header.

Reply:
I'll document it in the construction method as these callbacks can be used in multiple contexts. For example, this pattern is already followed for the task runner interface.